spark/src/main/scala/org/apache/comet/serde/QueryPlanSerde.scala

```diff
@@ -929,6 +929,19 @@ object QueryPlanSerde extends Logging with ShimQueryPlanSerde with CometExprShim
       binding: Boolean): Option[Expr] = {
     SQLConf.get
 
+    def convert(handler: CometExpressionSerde): Option[Expr] = {
+      handler match {
+        case _: IncompatExpr if !CometConf.COMET_EXPR_ALLOW_INCOMPATIBLE.get() =>
+          withInfo(
+            expr,
+            s"$expr is not fully compatible with Spark. To enable it anyway, set " +
```

Comment:
```suggestion
            s"$expr is not fully compatible with Spark, see details in Compatibility Guide. To enable it anyway, set " +
```

Reply:
I pushed a commit to update this by referencing a new `CometConf.COMPAT_GUIDE` variable with the link to the published documentation.